redi-connect-front/src/pages/app/me/Contacts.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { Content } from 'react-bulma-components'
+import FormInput from '../../../components/atoms/FormInput'
+import Editable from '../../../components/molecules/Editable'
+import { RedProfile } from '../../../types/RedProfile'
+import { connect } from 'react-redux'
+import { RootState } from '../../../redux/types'
+
+import {
+  profileSaveStart
+} from '../../../redux/user/actions'
+import * as Yup from 'yup'
+
+import { FormikValues, useFormik } from 'formik'
+
+export interface ContactsFormValues {
+  firstName: string
+  lastName: string
+  contactEmail: string
+  telephoneNumber: string
+}
+
+const validationSchema = Yup.object({
+  firstName: Yup.string()
+    .required()
+    .max(255),
+  lastName: Yup.string()
+    .required()
+    .max(255),
+  contactEmail: Yup.string()
+    .email()
+    .required()
+    .max(255)
+    .label('Contact email'),
+  telephoneNumber: Yup.string()
+    .max(255)
+    .label('Telephone number')
+})
+
+// props: FormikProps<AboutFormValues>
+const Contacts = ({ profile, profileSaveStart }: any) => {
```

Comment:
Need help with the TS Props definition? Let's make sure to avoid `any` in the final stage of this branch